.github/workflows/run-build.yml

```diff
@@ -0,0 +1,46 @@
+run the centos image produced by the builder with the repo files mapped in a volume
+runs create.py inside this container
+pushes the results to S3. This thus fully replaces cf.gov-ansible-build
```

Comment:
Does yaml allow free text like this? Shouldn't it be preceded by `#` commenting?

Reply:
hah! these were just my notes